DisplayOptionsPanel.c

```diff
@@ -116,8 +116,14 @@ DisplayOptionsPanel* DisplayOptionsPanel_new(Settings* settings, ScreenManager*
    Panel_add(super, (Object*) CheckItem_newByRef("Add guest time in CPU meter percentage", &(settings->accountGuestInCPUMeter)));
    Panel_add(super, (Object*) CheckItem_newByRef("Also show CPU percentage numerically", &(settings->showCPUUsage)));
    Panel_add(super, (Object*) CheckItem_newByRef("Also show CPU frequency", &(settings->showCPUFrequency)));
-   #ifdef HAVE_SENSORS_SENSORS_H
-   Panel_add(super, (Object*) CheckItem_newByRef("Also show CPU temperature (requires libsensors)", &(settings->showCPUTemperature)));
+   #ifdef BUILD_WITH_CPU_TEMP
+   Panel_add(super, (Object*) CheckItem_newByRef(
+   #ifdef HTOP_LINUX
+                                                 "Also show CPU temperature (requires libsensors)",
+   #elif defined(HTOP_FREEBSD)
+                                                 "Also show CPU temperature",
+   #endif
```

Comment:
If neither is defined, you'll get a peculiar compile error because of the missing parameter; I realise that the whole **point** is to have one or the other when `BUILD_WITH_CPU_TEMP` is set, but for future-proofing (e.g. OpenBSD support) you might have `#else #error Unsupported HTOP OS setting` or something like that.